### PR TITLE
Finite element solver input file support

### PR DESCRIPTION
**Pull request by sobolevnrm**
More work towards #16

### CODE REVIEW

File: apbs/test/test_input_file/test_calculate/test_finite_element.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    ],
+}
+GOOD_A_PRIORI_REFINEMNT = ["geometric", "uniform"]
+GOOD_BOUNDARY_CONDITIONS = [
+    "zero",
+    "single sphere",
+    "multiple sphere",
+    "focus foo",
+]
+GOOD_BEER = ["White Bluffs FID"]
```
Comment (intendo) [inline apbs/test/test_input_file/test_calculate/test_finite_element.py:46]: Was this a test to see if I would find something or is this really used?

**Review by intendo — APPROVED** [review]

File: apbs/test/test_input_file/test_calculate/test_finite_element.py (hunk 1 above)
Comment (intendo) [reply apbs/test/test_input_file/test_calculate/test_finite_element.py:46]: @sobolevnrm I would have gone with "Atomic Ale B-17".

Comment (sobolevnrm) [reply apbs/test/test_input_file/test_calculate/test_finite_element.py:46]: It was a test.  I guess you passed. :)  Want me to remove it?

Comment (intendo) [reply apbs/test/test_input_file/test_calculate/test_finite_element.py]: I removed it so we look more, professional?

**Comment by intendo** [comment]
I left the branch so you can keep working on it but you may want to do a pull to get the updates from Noah's work on the build system.

**Pull request merged**